extensions/file-manager/src/components/directory.tsx

```diff
@@ -45,6 +46,13 @@ export function Directory(props: { path: string; ignores: GitIgnoreHelper[]; ini
           return false;
         }
       }
+      if (
+        !preferences.showChflagsHiddenFiles &&
+        f.type == "file" &&
+        execSync(`ls -lO "${props.path}/${f.name}"`, { encoding: "utf8" }).toLowerCase().includes("hidden")
```

Comment:
Using `.includes("hidden")` causes issues if the username, or file name contains `hidden`. 

Instead, maybe use `stat -f%f {file}` and check for the bit 32768 (2^15).

See: https://discussions.apple.com/thread/2392066?sortBy=rank

I think that splitting by whitespace could also work, but I'm not sure if would get messy